resources/ci/buildArchitectureConfigurations.groovy

```diff
@@ -49,15 +55,24 @@ void runBuildConfigurationJob(
     )
 }
 
+/**
+ * Remove the branch prefix from the Connext version.
+ *
+ * @param repositoryBranch The rticonnextdds-examples branch to build.
+ * @returns The Connext version to build.
+ */
+String getVersionFromBranch(String repositoryBranch) {
+    return repositoryBranch - 'release/'
```

Comment:
I think that you can store just the version in the YAML file.

Reply:
and then you add the release/ instead of removing it.

Reply:
This is needed in order to know from the yaml, which of the branch names are release to pass it to the job.